Code/BasicFilters/src/sitkCastImageFilter-3.cxx

```diff
@@ -38,5 +36,4 @@ void CastImageFilter::RegisterMemberFactory3()
 
 }
 
-} // end namespace simple
 } // end namespace itk
```

Comment:
Should these end comments be changed to itk::simple?  And others to itk::simple::detail?

Reply:
Good check. I think the comments were needed before to keep track of the nested '}', so I think I'll remove them... *invoking sed*....